configs/ajio_local_config.properties

```diff
@@ -20,4 +20,5 @@ TARGET_ENVIRONMENT=prod
 LAUNCH_NAME_SUFFIX= on 'prod' Environment
 TEST_DATA_FILE=./src/test/resources/testData.json
 CLEANUP_DEVICE_BEFORE_STARTING_EXECUTION=false
-BROWSER_CONFIG_FILE=./configs/browser_config.json
\ No newline at end of file
+BROWSER_CONFIG_FILE=./configs/browser_config.json
+APPIUM_JS_PATH=/Users/mayank17.gupta/.appium/node_modules/appium/lib/main.js
```

Comment:
why is this required? Also, it cannot be a hardcoded path

Reply:
This APPIUM_JS_PATH config is required as part of ATD, without this test case will not run. Shall I remove my username and enter "Your system UserName" and then push the change. Please suggest.